maestro-cli/src/main/java/maestro/cli/App.kt

```diff
@@ -74,7 +79,10 @@ class App {
     @Option(names = ["--port"], hidden = true)
     var port: Int? = null
 
-    @Option(names = ["--device", "--udid"], description = ["(Optional) Device ID to run on explicitly, can be a comma separated list of IDs: --device \"Emulator_1,Emulator_2\" "])
+    @Option(
+        names = ["--device", "--udid"],
+        description = ["(Optional) Device ID to run on explicitly, can be a comma separated list of IDs: --device \"Emulator_1,Emulator_2\" "],
+    )
```

Comment:
Is this intentional change?

Reply:
yes. I just made it 2 lines instead of 1 to be more readable.

Reply:
👍